hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/FreezeServiceImpl.java

```diff
@@ -31,6 +31,8 @@
 public final class FreezeServiceImpl implements FreezeService {
     // special files will move to FileService
     public static final String UPGRADE_FILES_KEY = "SPECIAL_FILES";
+    public static final String UPGRADE_FILEID_KEY = "UPGRADE_FILEID";
```

Comment:
Nit: I guess normally we'd say `UPGRADE_FILE_ID_KEY`, so that `FILE_ID` rather than `FILEID`. In camel case we'd say `fileId`, emphasizing they are two words.

Reply:
I renamed the constant to `UPGRADE_FILE_ID_KEY`. Note that in the protobufs it's `FileID` (with a capital D) so I've followed that convention and called it `fileID` when in camel case. https://hashgraph.github.io/hedera-protobufs/#proto.FileID